src/test/java/org/gridsuite/modification/server/utils/NetworkForDeleteVoltageLevelOnLine.java

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
This class is never used, and is a duplicate of NetworkCreation.createForDeleteVoltageLevelOnLine() below.
Do be deleted ?